cmd/devenv/apps/update/update.go

```diff
@@ -120,7 +118,11 @@ func (o *Options) Run(ctx context.Context) error {
 		return err
 	}
 
+	// devenv apps update should not take any arguments and should use devenv apps deploy instead
 	if o.AppName != "" {
+		o.log.Warn("DEPRECATED: Use `devenv apps deploy` instead to update a services version")
+		time.Sleep(1 * time.Second)
```

Comment:
```suggestion
		time.Sleep(5 * time.Second)
```

I like to make intentional sleeps a little longer :D